charge-holy-weapon.lic

```diff
@@ -28,19 +28,19 @@ class HolyWeapon
   def check_charge_level(hw_settings)
     @em.wield_weapon(hw_settings['weapon_name'])
 
-    case bput("look my #{hw_settings['weapon_name']}", 'barely detectable', 'flickering', 'barely glowing', 'faintly', 'shining', 'emanating', 'blinding')
+    case DRC.bput("look my #{hw_settings['weapon_name']}", 'barely detectable', 'flickering', 'barely glowing', 'faintly', 'shining', 'emanating', 'blinding')
     when 'barely detectable', 'flickering'
       complete_ritual(hw_settings)
     when 'barely glowing'
-      message "200 estimated charges remain, skipping..."
+      DRC.message("200 estimated charges remain, skipping...")
```

Comment:
Per [Madigan](https://discord.com/channels/745675889622384681/745678209449853049/796807519934480435), would like way to configure at which power level you want to recharge your weapon. 

> The only change I would recommend is removing the two lower "checks".  Once you get down below 200 strikes, you really need to re-charge. I have manually changed that for my version, but that is probably a global change that would be useful IMO.

Not sure best way to represent that in the yaml config. Since we don't know the _exact_ number of strikes, I don't think asking the player to specify a number makes sense. We could use "power level" on scale of 1-8 as documented on the wiki that is the threshold at or below you want to recharge your weapon.

For example:
```yaml
holy_weapon:
  weapon_name: greatsword
  icon_name: holy icon
  icon_container: backpack
  icon_room: 1234
  # Indicate the power level when you want to recharge your holy weapon.
  # Specify a number between 1 (when no charges) and 8 (all or nearly all charges).
  # See also https://elanthipedia.play.net/Holy_weapon#Power_Levels
  power_level_threshold: 3
```

Reply:
Will address this new feature in a separate pull request